scripts/core/adapter.yml

```
@@ -73,12 +73,19 @@ params:
           [in] Adapter handle to retain
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Get the last adapter specific error."
+desc: "Get the last adapter specific warning or error."
 details: |
     To be used after another entry-point has returned
     $X_RESULT_ERROR_ADAPTER_SPECIFIC in order to retrieve a message describing
     the circumstances of the underlying driver error and the error code
     returned by the failed driver entry-point.
+  
+    * If `pError` is $X_RESULT_SUCCESS, $X_RESULT_ERROR_ADAPTER_SPECIFIC
+    represents a warning instead. This means that the entry-point call did not
+    fail. However it might not have behaved as expected.
+  
+    * Using $X_RESULT_ERROR_ADAPTER_SPECIFIC to emit warnings is an optional
+    feature. Its usage is left at the discretion of adapter maintainers.
```

Indenting the bullet points makes them stand out more to the eye, also matches the surrounding style.

```suggestion
    * If `pError` is $X_RESULT_SUCCESS, $X_RESULT_ERROR_ADAPTER_SPECIFIC
      represents a warning instead. This means that the entry-point call did not
      fail. However it might not have behaved as expected.
  
    * Using $X_RESULT_ERROR_ADAPTER_SPECIFIC to emit warnings is an optional
      feature. Its usage is left at the discretion of adapter maintainers.
```

---

Done :+1: